### PR TITLE
Add changelog with 3.0.0rc2

### PR DESCRIPTION
**Pull request by hhaAndroid**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily get feedback. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation

Please describe the motivation of this PR and the goal you want to achieve through this PR.

## Modification

Please briefly describe what modification is made in this PR.

## BC-breaking (Optional)

Does the modification introduce changes that break the backward-compatibility of the downstream repos?
If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.

## Use cases (Optional)

If this PR introduces a new feature, it is better to list some use cases here, and update the documentation.

## Checklist

1. Pre-commit or other linting tools are used to fix the potential lint issues.
2. The modification is covered by complete unit tests. If not, please add more unit test to ensure the correctness.
3. If the modification has potential influence on downstream projects, this PR should be tested with downstream projects, like MMDet or MMCls.
4. The documentation has been modified accordingly, like docstring or example tutorials.

### CODE REVIEW

File: README_zh-CN.md [hunk 1]
```diff
@@ -74,10 +74,10 @@ MMDetection 是一个基于 PyTorch 的目标检测开源工具箱。它是 [Ope
 
 ## 最新进展
 
-**v3.0.0rc1** 版本已经在 2022.9.26 发布：
+**v3.0.0rc2** 版本已经在 2022.10.21 发布：
 
-- 发布高精度低延时单阶段目标检测算法 [RTMDet](configs/rtmdet)。
-- 重构 Anchor Head 和 Base Head 以支持 `box type`。
+- RTMDet 骨干网络支持 [ImageNet 预训练](<(configs/rtmdet/cspnext_imagenet_pretrain)>)。
```
Comment (RangiLyu) [inline README_zh-CN.md:79]:
```suggestion
- RTMDet 骨干网络支持 [训练分类任务](<(configs/rtmdet/cspnext_imagenet_pretrain)>)。
```

Comment (ZwwWayne) [reply README_zh-CN.md:79]: 一个检测器支持训练分类任务也有点奇怪，要不改成：支持在 ImageNet 上预训练 RTMDet 的骨干网络

File: docs/en/notes/changelog.md [hunk 2]
```diff
@@ -1,5 +1,53 @@
...
+- Fix `CircleCI` and `readthedoc` build failed (#8980, #8963)
+- Fix bitmap mask translate when `out_shape` is different (#8993)
+- Fix inconsistency in `Conv2d` weight channels (#8948)
+- Fix bugs when plotting loss curve by analyze_logs.py (#8944)
+- Fix type change of labels in `albumentations` (#9074)
+- Fix some docs and types error (#8818)
+
+#### Improvements
+
+- Refactored standard roi head with `box type` (#8658)
```
Comment (ZwwWayne) [inline docs/en/notes/changelog.md:27]: Refactored -> Refactor

File: docs/en/notes/changelog.md [hunk 3]
```diff
@@ -1,5 +1,53 @@
...
+- Support mask concatenate in `BitmapMasks` and `PolygonMasks` (#9006)
```
Comment (ZwwWayne) [inline docs/en/notes/changelog.md:28]: concatenation

Comment (ZwwWayne) [reply docs/en/notes/changelog.md:28]: We should also tell users that we bump the dependency of the version of mmengine

Comment (hhaAndroid) [reply docs/en/notes/changelog.md:28]: There is no change in the `mmengine` version. Just changed the upper limit.

File: docs/en/notes/changelog.md [hunk 4]
```diff
@@ -1,5 +1,56 @@
...
+#### Improvements
+
+- Refactor standard roi head with `box type` (#8658)
+- Support mask concatenation in `BitmapMasks` and `PolygonMasks` (#9006)
+- Update pytorch and dependencies' version in dockerfile (#8845)
+- Update `robustness_eval.py` and `print_config` (#8452)
+- Make compatible with `ConfigDict` and `dict` in `dense_heads` (#8942)
+- Support log coco metric copypaste (#9012)
+- Remove `Normalize` transform (#8913)
+- Jitter instance color (#8988)
```
Comment (ZwwWayne) [inline docs/en/notes/changelog.md:36]: Support jittering the color of different instances of the same class

File: docs/en/notes/changelog.md [hunk 5]
```diff
@@ -1,5 +1,56 @@
...
+- Fix wrong arguments of `OpenImageMetrics` in the config (#9061)
+
+#### Improvements
+
+- Refactor standard roi head with `box type` (#8658)
+- Support mask concatenation in `BitmapMasks` and `PolygonMasks` (#9006)
+- Update pytorch and dependencies' version in dockerfile (#8845)
+- Update `robustness_eval.py` and `print_config` (#8452)
+- Make compatible with `ConfigDict` and `dict` in `dense_heads` (#8942)
+- Support log coco metric copypaste (#9012)
```
Comment (ZwwWayne) [inline docs/en/notes/changelog.md:34]: Support logging

File: docs/en/notes/changelog.md [hunk 6]
```diff
@@ -1,5 +1,56 @@
...
+- Fix type change of labels in `albumentations` (#9074)
+- Fix some docs and types error (#8818)
+- Update memory occupation of RTMDet in metafile (#9098)
+- Fix wrong arguments of `OpenImageMetrics` in the config (#9061)
+
+#### Improvements
+
+- Refactor standard roi head with `box type` (#8658)
+- Support mask concatenation in `BitmapMasks` and `PolygonMasks` (#9006)
+- Update pytorch and dependencies' version in dockerfile (#8845)
```
Comment (ZwwWayne) [inline docs/en/notes/changelog.md:31]: PyTorch

**Review by ZwwWayne — APPROVED** [review]

**Pull request merged**